server/bin/test_data_importer.py

```diff
@@ -0,0 +1,706 @@
+                    large_quantity = small_quantity * 10 if small_quantity > 0 else 10
+                else:
+                    small_quantity = 5
+                    large_quantity = 50
+
+                start_date = (now + datetime.timedelta(days=-5)).isoformat()
+                end_date = (now + datetime.timedelta(days=360)).isoformat()
+
+                # Small quantity pool
+                pools.append({
+                    'product_id': product['id'],
+                    'quantity': small_quantity,
+                    'contract_number': 0,
+                    'account_number': account_number,
+                    'order_number': order_number,
+                    'start_date': start_date,
+                    'end_date': end_date
+                })
+
+                # Large quantity pool
+                pools.append({
+                    'product_id': product['id'],
+                    'quantity': large_quantity,
+                    'contract_number': 0,
+                    'account_number': account_number,
+                    'order_number': order_number,
+                    'start_date': start_date,
+                    'end_date': end_date
+                })
+
+                # Future pool
+                pools.append({
+                    'product_id': product['id'],
+                    'quantity': large_quantity,
+                    'contract_number': 0,
+                    'account_number': account_number,
+                    'order_number': order_number,
+                    'start_date': (now + datetime.timedelta(days=30)).isoformat(),
+                    'end_date': end_date
+                })
+
+            else:
+                # Product is an engineering product. Flag the pid/owner combo so we can fetch the cert later
+                engineering_products[product['id']] = owner_key
+
+        log.info("  Creating %d subscription(s) for owner %s..." % (len(pools), owner_key))
+        for pool in pools:
+            cp.create_pool(owner_key, pool)
+
+        # Refresh (if necessary)
+        if cp.hosted_test:
+            log.info("  Refreshing owner: %s" % (owner_key))
+            cp.refresh_pools(owner_key)
+
+
+    # Create activation keys
+    for owner_key in owners:
+        pools = cp.list_pools(owner_key)
+        activation_keys.extend([(owner_key, "%s-%s-key-%s" % (owner_key, pool['productId'], pool['id']), pool) for pool in pools])
+
+    if activation_keys:
+        print('')
+        log.info("Creating %d activation keys..." % (len(activation_keys)))
+        for owner_key, key_name, pool in activation_keys:
+            log.info("  Activation key: %s [owner: %s, pool: %s]" % (key_name, owner_key, pool['id'] if pool is not None else 'n/a'))
+
+            key = cp.create_activation_key(owner_key, {'name': key_name})
+            if pool is not None:
+                cp.add_pool_to_activation_key(key['id'], pool['id'])
+
+
+    # Fetch entitlements/certs for engineering products
```

Comment:
IMHO these certificates are not entitlement certificates, but these certificates are `productid` certificates.

Reply:
Correct. The comment is wrong here.